charts/pvc-autoresizer/Chart.yaml

```diff
@@ -18,10 +18,18 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.1
+version: 0.6.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: 0.6.0
+
+annotations:
+  artifacthub.io/images: |
+    - name: pvc-autoresizer
+      image: ghcr.io/topolvm/pvc-autoresizer:0.6.0
```

Comment:
We have a procedure to update the Chart.yaml.
https://github.com/topolvm/pvc-autoresizer/blob/main/RELEASE.md?plain=1#L132
Could you update the document?

Reply:
updated.